packages/react-composites/src/composites/ChatComposite/ChatComposite.tsx

```diff
@@ -28,9 +28,20 @@ export type ChatCompositeProps = {
 };
 
 export type ChatOptions = {
-  sendBoxMaxLength?: number; // Limit max send box length, when change viewport size
-  // messagesPerPage?: number; // Number of messages per page - smaller for better perf
-  // supportNewline: boolean; // Whether to support new line (shift+enter) in textArea, disable until ACS backend supports line switch
+  /** Choose to show the participant pane */
+  showParticipantPane?: boolean;
+  /** Set a max width of the send box */ // TODO: we should remove this.
+  sendBoxMaxLength?: number;
+  /**
```

Comment:
nitty: Just delete these commented out options?

There's always history. Dead code is a drag.

Reply:
Agree, removed and filed bugs in their place
https://skype.visualstudio.com/SPOOL/_workitems/edit/2508474
https://skype.visualstudio.com/SPOOL/_workitems/edit/2508476